lib/Common/Core/ConfigParser.cpp

```diff
@@ -337,19 +337,70 @@ void ConfigParser::ParseConfig(HANDLE hmod, CmdLineArgsParser &parser)
         {
             return;
         }
+
         if (_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")) != 0 || configFile == nullptr)
         {
             return;
         }
     }
+#else
+    // Two-pathed for a couple reasons
+    // 1. PAL doesn't like the ccs option passed in.
```

Comment:
I'm missing something- are you saying that this line of code works on Linux:
`_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")`

How does it work? I see the following use of mode in the PAL implementation of PAL_fopen:
```
PAL_fopen(const char * fileName, const char * mode)
{
    PAL_FILE *f = NULL;
    LPSTR supported = NULL;
    LPSTR UnixFileName = NULL;
    struct stat stat_data;
    BOOL bTextMode = TRUE;

    PERF_ENTRY(fopen);
    ENTRY("fopen ( fileName=%p (%s) mode=%p (%s))\n", fileName, fileName, mode , mode );

    _ASSERTE(fileName != NULL);
    _ASSERTE(mode != NULL);

    if ( *mode == 'r' || *mode == 'w' || *mode == 'a' )
    {
        supported = MapFileOpenModes( (char*)mode,&bTextMode);
```

This defers to MapFileOpenModes, and when I look at that function, I don't see any handling of `ccs=Unicode`. 

This matches what I see when I run the code too (that calling _wfopen_s with `ccs=Unicode` fails but calling it without passes. Can you help me understand what I missed?